crates/neon-runtime/src/napi/call.rs

```diff
@@ -28,7 +29,17 @@ pub unsafe extern "C" fn get_isolate(_info: FunctionCallbackInfo) -> Env { unimp
 // FIXME: Remove. This will never be implemented
 pub unsafe extern "C" fn current_isolate() -> Env { panic!("current_isolate won't be implemented in n-api") }
 
-pub unsafe extern "C" fn is_construct(_info: FunctionCallbackInfo) -> bool { unimplemented!() }
+pub unsafe extern "C" fn is_construct(env: Env, info: FunctionCallbackInfo) -> bool {
+    let mut target: MaybeUninit<napi::napi_value> = MaybeUninit::zeroed();
```

Comment:
We use the type alias `Local` in most neon-runtime functions instead of `napi_value`. When N-API is the only runtime that doesn't make much sense anymore, but for now perhaps it's best to use `Local` for consistency? 😄